testing.go

```diff
@@ -20,6 +20,12 @@ var errAlreadyInProgress = regexp.MustCompile(`removal of container .* is alread
 // This is a function designed to be used in your test, when Docker is not mandatory for CI/CD.
 // In this way tests that depend on Testcontainers won't run if the provider is provisioned correctly.
 func SkipIfProviderIsNotHealthy(t *testing.T) {
+	defer func() {
+		if r := recover(); r != nil {
+			t.Skipf("Recovered from panic. Docker is not running. TestContainers can't perform is work without it")
```

Comment:
Typo: can you apply it to the rest of occurrences in the file 🙏 ?
```suggestion
			t.Skipf("Recovered from panic. Docker is not running. Testcontainers can't perform is work without it")
```

Reply:
I would also include the value of `r` as there could be a few different cases and this would disambiguate them.